src/pages/management/configuration-api/v3.5/index.mdx

```diff
@@ -2909,6 +2909,65 @@ curl -X GET \
+
+Limited resources in LiveChat:
+
+| Resource Name          | Notes                                                                                                                                |
+| ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------ |
+| `agents`               | The max number of active agents, including bots                                                                                      |
+| `groups`               | The max number of groups                                                                                                             |
+| `banned`               | The max number of banned customers                                                                                                   |
+| `queue`                | The max number of chats waiting in the queue (with [Messaging Mode](/messaging#messaging-mode) disabled)                             |
+| `queue_continuous`     | The max number of chats waiting in the queue (with [Messaging Mode](/messaging#messaging-mode) enabled)                              |
+| `groups_per_agent`     | The max number of groups an agent can be assigned to                                                                                 |
+| `access_rules`         | The max number of [Access Rules](/management/configuration-api#auto-access)                                                          |
+| `group_chooser_groups` | The max number of groups available in the [pre-chat survey](https://www.livechat.com/help/dividing-live-chat-by-group/#target-field) |
+
+#### Specifics
+
+|                 |                                                                                       |
+| --------------- | ------------------------------------------------------------------------------------- |
+| HTTP Method     | **GET**                                                                               |
+| Method URL      | `https://api.livechatinc.com/v3.5/configuration/action/check_product_limits_for_plan` |
+| Required scopes | `subscription.lc:rw`                                                                  |
+
+#### Request
+
+| Parameter | Required | Data type | Notes                                                                |
+| --------- | -------- | --------- | -------------------------------------------------------------------- |
+| `plan`    | yes      | `string`  | Possible values: `starter`, `team`, `enterprise`, `enterpriseplus`   |
+
+</Text>
+
+<Code>
+
+<CodeSample path={'REQUEST'}>
+
+```shell
+curl -X POST \
+  https://api.livechatinc.com/v3.5/configuration/action/check_product_limits_for_plan \
+  -H 'Authorization: Bearer <your_access_token>' \
+  -H 'Content-Type: application/json' \
+  -d '{
+        "plan": "starter"
+      }'
+```
+
+</CodeSample>
+
+<CodeResponse type="configuration" version="v3.5" json="checkProductLimitsForPlan"/>
```

Comment:
The response is not rendered. This json should also be exported in `payloads/management/v3.5/configuration-api/index.js`

Also - is the method name get or check?

Reply:
It should be "check*", fixed